frontend/app/src/components/accounts/AccountGroupHeader.vue

```diff
@@ -37,7 +37,7 @@ const label = computed<string | undefined>(() => get(xpub).label);
 const xpubTags = computed<string[] | undefined>(() => get(xpub).tags);
 
 const displayXpub = computed<string>(() =>
-  scrambleAddress(truncateAddress(get(xpub).data.xpub, truncationPoints[get(breakpoint)] ?? 4)),
+  truncateAddress(get(xpub).data.xpub, truncationPoints[get(breakpoint)] ?? 4),
```

Comment:
Doesn't that mean that the `xpub` won't be scrambled anymore?

Reply:
my bad, updated